app/src/main/java/ca/bc/gov/bchealth/compose/component/QuickAccessTileItemUI.kt

```diff
@@ -0,0 +1,97 @@
+package ca.bc.gov.bchealth.compose.component
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.material.Card
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import androidx.constraintlayout.compose.ConstraintLayout
+import androidx.constraintlayout.compose.Dimension
+import ca.bc.gov.bchealth.R
+import ca.bc.gov.bchealth.compose.BasePreview
+import ca.bc.gov.bchealth.compose.theme.HealthGatewayTheme
+
+@Composable
+fun QuickAccessTileItemUI(
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    icon: Painter,
+    title: String
+) {
+    Card(
+        modifier = modifier
+            .height(120.dp)
+            .clickable { onClick() },
+        elevation = 15.dp,
+        backgroundColor = MaterialTheme.colors.background
+    ) {
+
+        ConstraintLayout(
+            modifier = modifier
+                .fillMaxSize()
+        ) {
+
+            val (cardIcon, cardTitle, arrow) = createRefs()
+
+            Image(
+                modifier = Modifier
+                    .size(32.dp)
+                    .constrainAs(cardIcon) {
+                        start.linkTo(parent.start, 16.dp)
+                        top.linkTo(parent.top, 16.dp)
+                    },
+                painter = icon,
+                contentDescription = title
```

Comment:
The content description here is the same as the Text below, please remove it or update with an appropriate content

Reply:
done